CONTRIBUTING.md

```diff
@@ -98,24 +99,33 @@ Contributors may provide database identifiers associated with each member of the
 
 mockrobiota does not host raw data files (e.g., sequencing files). All sequencing data and other raw data files must be deposited on public, external websites. Stable, public depositories are preferred, but this requirement is not enforced by mockrobiota. mockrobiota ensures that valid, accessible links are provided in the dataset metadata (if not, integrity checks will fail and your dataset will not be accepted), but does not manage these external resources and can not guarantee the validity of raw data that are contributed by outside users. When preparing raw data for linking to mockrobiota datasets, please observe the following regulations:
 
-1. All raw sequence data should be deposited in .fastq format and archived using standard compression formats, e.g., .gz or .zip.
+1. All raw sequence data must be deposited in .fastq format and archived using gzip (required file extension: .gz)
 2. Mock community datasets that contain multiple samples must be provided in non-demultiplexed files (i.e., one file per read direction per sequencing run, containing multiple uniquely barcoded samples).
-3. Index/barcode sequences must be provided as a separate .fastq file. If QUAL scores do not exist for these reads, please note this in the human-readable-description field of dataset-metadata.tsv for that dataset.
+3. Index/barcode sequences must be provided as a separate .fastq file if sequence reads are multiplexed. If QUAL scores do not exist for these reads, please note this in the description field of ``README.md`` for that dataset. This file is optional **only** if the mock community contains one sample and the sequence reads are demultiplexed fastq files.
 4. Reverse sequencing reads are accepted, but not required. Forward and reverse reads should be submitted as separate files, not as joined reads.
 5. All raw data must conform to the following naming conventions:
     - mock-forward-read.fastq.gz
     - mock-reverse-read.fastq.gz (if applicable)
-    - mock-index-read.fastq.gz
+    - mock-index-read.fastq.gz (if applicable)
+
+
+## Source files
+The ``source`` directory for each community contains various information defined by the contributor. ``source/taxonomy.tsv``, described above, is required, as it should ideally contain strain information that may not be preserved in the database-specific ``expected-taxonomy.tsv`` files. Additionally, other files may be relevant, such as full-length 16S rRNA (or other marker-gene) sequences that the user sequenced from each strain. Naming conventions are not enforced for these optional files, but try to use short, descriptive names. As github enforces size limits on repositories, only small files are accepted ``source``, and may be removed by the mockrobiota developers at their personal discretion without notice to the contributor. Don't even try uploading whole genome sequences.
```

Comment:
* *github* -> *GitHub*

* *only small files are accepted ``source``* -> *only small files can be provided in ``source``*

* *Don't even try uploading whole genome sequences.* -> * Whole genome sequences will definitely be too large to include.*